01-js-intro-and-jquery/01/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
```

Comment:
Just to have in mind for next time, I think the idea was to solve everything using the same files (no need to create a file to follow each point of the exercises).